insights/src/main/resources/module/main/module.xml

```diff
@@ -0,0 +1,20 @@
+<module xmlns="urn:jboss:module:1.0" name="org.jboss.as.insights">
```

Comment:
this module.xml doesn't really have any meaning on this location, probably safe to remove.
You have it in feature pack